packages/backend-common/src/tokens/ServerTokenManager.ts

```diff
@@ -37,20 +30,42 @@ export class ServerTokenManager implements TokenManager {
   private readonly verificationKeys: JWKS.KeyStore;
   private readonly signingKey: JWK.Key;
 
-  static noop(): TokenManager {
-    return new NoopTokenManager();
+  static default(options: { config: Config; logger: Logger }) {
```

Comment:
The replacement of .noop with .default implies a couple of other changes:

1. Update the [backend index in create-app](https://github.com/backstage/backstage/blob/master/packages/create-app/templates/default-app/packages/backend/src/index.ts#L41) to create a default token manager (and perhaps add a comment above to let new integrators know they'll need to configure a secret and set the appropriate environment variable before deploying.
2. Update the [backend-to-backend auth docs](https://github.com/backstage/backstage/blob/master/docs/tutorials/backend-to-backend-auth.md).

Reply:
> This was part of the original PR, right?

It's a bit sneaky - there's packages/backend/src/index.ts, which this PR _does_ correctly update, but there's also packages/create-app/templates/default-app/packages/backend/src/index.ts, which will need to be updated too since it currently uses ServerTokenManager.noop(). Kinda takes us back to the discussion above about opting everyone into backend-to-backend auth 😅.

Reply:
Updated the create-app file, and resolving this thread to keep discussion about forced opt-in in the other thread